### ISSUE TITLE
Improve logging

### ISSUE THREAD

**Issue by ckoopmann** [IndexCoop/index-arb-bots#5]
Currently logs on tenderly get truncated since we have too many of them. (see screenshot).

We should fix that either by:
1. Somehow increasing the max amount of logs (if there is any way in tenderly to achieve that)
2. Reduce the amount of logging
3. Use a third party logging collection service (such as datadog)

<img width="1010" alt="image" src="https://github.com/IndexCoop/index-arb-bots/assets/15629702/709888d5-440f-4050-b0a1-b162e95d8305">

**Comment by janndriessen** [comment]
Good idea using a third party service 👌 @ckoopmann I guess this might be the way cause I think there was some limitation by tenderly but can't find it in the docs any longer.

When you're logged in you can use the support chat, they were usually really helpful.
